apps/graphql/src/server.js

```diff
@@ -17,9 +18,19 @@ const server = new ApolloServer({
     //   - tokens and user identities are per request
     //   - dataloaders use Map internally (not LRU) and they would otherwise
     //     grow indefinitely because the Map content is not garbage collected
+    const [language, territory] = parseAcceptLanguage(
+      req.headers['accept-language'],
+    );
+
+    const locale = {
+      language,
+      territory: territory,
+    };
+
     return {
       clientIP: requestIp.getClientIp(req),
       ...createContext(),
+      locale,
```

Comment:
We have a function called createContext, I think we should manage all logic there, instead of some of the logic in that function, and some logic in `server.js`